redis-snapshot-replicator/functions/create_snapshot.py

```diff
@@ -0,0 +1,17 @@
+import boto3  
+import datetime 
+import os
+
+instances = os.environ['DB_INSTANCES']
+iam = boto3.client('iam')
+print('Loading function')
+
+def lambda_handler(event, context):  
+    source = boto3.client('elasticache')
+
+    for instance in instances.split(','):
+        now = datetime.datetime.now()
+        db_snapshot_name = now.strftime('%Y-%m-%d-%H-%M')
+        response = source.create_snapshot(CacheClusterId=instance,SnapshotName='snapshot-'+db_snapshot_name)
```

Comment:
Does this need to have any additional event logging to know the outcome of the Lambda  call?

Reply:
At this point I have not added any validation, I can add this if we go on with this module